lib/game_statistics.rb

```diff
@@ -1,26 +1,34 @@
-require_relative "stat_tracker"
+require_relative "./stat_tracker"
+require_relative "./stat_helper"
 
-class GameStatistics
-  # def self.from_csv(files)
-  #   require 'pry'; binding.pry
-  # end
+class GameStatistics < StatHelper
 
-  # def initialize(locations)
-  # end
-# def highest_total_score
-  #   method
-  # end
+  def initialize(files)
+    super
```

Comment:
what is "super"?

Reply:
it is a method that says bypass this method in the look-up chain and go straight to the parent class as use the same method as named in the parent(super) class. The child is the sub class, in this case which is the GameStatistics class.